grails-app/views/opus/_report.gsp

```diff
@@ -84,4 +87,84 @@
     </div>
 
     </script>
+
+    <script type="text/ng-template" id="mostRecentChange.html">
+    <div class="">
+        <label class="control-label">Show edited profiles from:</label>
+
+        <div class="btn-group" role="group" aria-label="List most recent changes with the following options.">
+            <button type="button" class="btn btn-default" ng-repeat="period in reportCtrl.periods"
+                    ng-click="reportCtrl.setPeriod(period)"
+                    ng-class="{active: reportCtrl.selectedPeriod.id == period.id}">{{ period.name }}</button>
+        </div>
+    </div>
+
+    <div class="well top-buffer" ng-show="reportCtrl.selectedPeriod.id == 'custom'">
+        <h4>Choose start and end dates</h4>
```

Comment:
Make this text smaller (perhaps just <p> instead of a header?), and change to "Select a date range"